components/config/src/lib.rs

```diff
@@ -38,7 +38,7 @@ pub struct Config {
     /// Description of the site
     pub description: Option<String>,
     /// The language used in the site. Defaults to "en"
-    pub language_code: Option<String>,
+    pub default_language: Option<String>,
```

Comment:
can this be deprecated first? ie leave `language_code` for now and add `default_language` but print a deprecation warning if `language_code` is set

Reply:
Note: next gutenberg version has breaking changes so it's fine